src/FSharp.AWS.DynamoDB/Expression/ExprCommon.fs

```diff
@@ -96,14 +96,13 @@ type AttributeId =
                 | :? string as f -> FField f
                 | :? int as i ->
                     if i < 0 then
-                        raise <| ArgumentOutOfRangeException()
+                        raise <| ArgumentOutOfRangeException ()
```

Comment:
no space before ctor args

Reply:
fantomas seems limited here - it will pick it up if you use `new` but restoring for 'upper case invocations' means DUs with a tuple will end up with no space

Reply:
would take no space before DU ctor and RU deconstruction over having methods and other ctors all uglified
as noted in mainline, we can ask nojaf if its our only beef in the end